bower.json

```diff
@@ -16,4 +16,4 @@
   "resolutions": {
     "route-recognizer": "^0.2.7"
   }
-}
+}
```

Comment:
I think your config is chopping off trailing line endings. I'm pretty sure this violates our `editorconfig` but not sure.